lib/beaker/dsl/install_utils/foss_utils.rb

```diff
@@ -1107,6 +1107,10 @@ def install_puppet_agent_dev_repo_on( hosts, opts )
             onhost_copy_base = opts[:copy_dir_external]
 
             case variant
+            when /^eos/
```

Comment:
yeah, I agree, helpers here would be way better.  I'm down with fixing these up as a policy going forward.